frontend/integration-tests/views/catalog.view.ts

```diff
@@ -6,6 +6,8 @@ export const pageNumberItemsHeading = $('.co-catalog-page__num-items');
 export const pageHeadingNumberOfItems = () => pageNumberItemsHeading.getText()
   .then(text => parseInt(text.substring(0, text.indexOf(' items')), 10));
 export const catalogDetailsLoaded = () => browser.wait(until.presenceOf($('.modal-content')), 10000).then(() => browser.sleep(1000));
-export const createServiceInstanceButton = $('.co-catalog-page__overlay-create');
+export const createResourceButton = $('.co-catalog-page__overlay-create');
 export const createServiceInstanceForm = $('.co-create-service-instance');
 export const createServiceBindingButton = $('.co-well').element(by.buttonText('Create Service Binding'));
+export const instantiateTemplateFrom = $('.co-instantiate-template-form');
+export const resourceItem = (resourceType: string, resourceName: string) => $(`.co-resource-item>.co-m-resource-icon[title=${resourceType}]+.co-resource-item__resource-name[href*=${resourceName}]`);
```

Comment:
Add a new `data-test-id` if needed. This selector looks fragile.

Reply:
I would give this a more specific name like `templateInstanceResource`

Reply:
> Add a new `data-test-id` if needed. This selector looks fragile.

It would be good if we can add `data-test-id`, can we?